examples/plasticity/WellboreProblem_PoroDruckerPrager.xml

```diff
@@ -0,0 +1,286 @@
+<!-- SPHINX_WELLBORE_ELEMENTREGIONS_END -->
+
+<!-- SPHINX_WELLBORE_BC -->
+<FieldSpecifications>
+  <FieldSpecification 
+    name="stressXX"
+    initialCondition="1"
+    setNames="{all}"
+    objectPath="ElementRegions/Omega/cb1"
+    fieldName="rock_stress"
+    component="0"
+    scale="-9.0e6"
+  />
+
+  <FieldSpecification 
+    name="stressYY"
+    initialCondition="1"
+    setNames="{all}"
+    objectPath="ElementRegions/Omega/cb1"
+    fieldName="rock_stress"
+    component="1"
+    scale="-11.0e6"
+  />
+
+  <FieldSpecification 
+    name="stressZZ"
+    initialCondition="1"
+    setNames="{all}"
+    objectPath="ElementRegions/Omega/cb1"
+    fieldName="rock_stress"
+    component="2"
+    scale="-12.0e6"
+  />
+
+  <FieldSpecification 
+    name="initialPressure"
+    initialCondition="1"
+    setNames="{all}"
+    objectPath="ElementRegions/Omega/cb1"
+    fieldName="pressure"
+    scale="0e6"
+  />
+
+  <FieldSpecification 
+    name="xconstraint"    
+    objectPath="nodeManager" 
+    fieldName="TotalDisplacement" 
+    component="0" 
+    scale="0.0" 
+    setNames="{xneg, xpos}"
+  />
+
+  <FieldSpecification 
+    name="yconstraint"    
+    objectPath="nodeManager" 
+    fieldName="TotalDisplacement" 
+    component="1" 
+    scale="0.0" 
+    setNames="{tneg, tpos, ypos}"
+  />
+
+  <FieldSpecification 
+    name="zconstraint" 
+    objectPath="nodeManager" 
+    fieldName="TotalDisplacement" 
+    component="2" 
+    scale="0.0" 
+    setNames="{zneg, zpos}"
+  />
+
+  <Traction 
+    name="InnerMechanicalLoad"
+    setNames="{ rneg }"
+    objectPath="faceManager"
+    scale="-10.0e6"
+    tractionType="normal"
+    functionName="timeFunction"     
+  />
+
+  <FieldSpecification 
+    name="InnerFluidLoad"
+    setNames="{ rneg }"
+    objectPath="faceManager"
+    fieldName="pressure"
+    scale="10e6"
+    functionName="timeFunction"   
+  />
+</FieldSpecifications>
+<!-- SPHINX_WELLBORE_BC_END -->
+
+<!-- SPHINX_WELLBORE_OUTPUT -->
+<Outputs>
+  <Silo 
+    name="siloOutput"
+    parallelThreads="1"
+    plotFileRoot="plot"
+  />
+</Outputs>
+<!-- SPHINX_WELLBORE_OUTPUT_END -->
+
+<!-- SPHINX_WELLBORE_POROELASTICSOLVER -->
+<Solvers gravityVector="{0.0, 0.0, 0.0}">
+<SinglePhasePoromechanics 
+  name="PoroelasticitySolver" 
+  solidSolverName="mechanicsSolver"
+  fluidSolverName="SinglePhaseFlowSolver"
+  porousMaterialNames="{porousRock}"
+  logLevel="1"
+  discretization="FE1"
+  targetRegions="{Omega}">
+  <LinearSolverParameters 
+    solverType="direct"
+    directParallel="0"
+    logLevel="0"
+  />
+  <NonlinearSolverParameters
+    newtonMaxIter = "40"
+  />
+</SinglePhasePoromechanics>
+<!-- SPHINX_WELLBORE_POROELASTICSOLVER_END -->
+
+<!-- SPHINX_WELLBORE_MECHANICALSOLVER -->
+<SolidMechanics_LagrangianFEM 
+  name="mechanicsSolver" 
+  timeIntegrationOption="QuasiStatic"
+  logLevel="1"
+  discretization="FE1"
+  targetRegions="{ Omega }"
+  solidMaterialNames="{ rock }">
+  <LinearSolverParameters 
```

Comment:
It would be good to run our new XML validate script over this file.  It is not clear to me this option block will actually be used.  It could certainly be deleted as it is set to the default.

Reply:
You're right and these unused block will be deleted.